src/main/resources/application.yml

```diff
@@ -33,6 +33,20 @@ spring:
     url: 'jdbc:h2:mem:restricted-patients-api-db;MODE=PostgreSQL'
     continue-on-error: true
 
+  security:
+    oauth2:
+      client:
+        registration:
+          manage-adjudications-api:
+            provider: hmpps-auth
+            client-id: ${manage.adjudications.api.client.id}
```

Comment:
Need to make sure these are defined in the k8s secrets section. Not sure it will start without those in place.